lego/apps/events/models.py

```diff
@@ -50,7 +50,7 @@ class Event(Content, BasisModel, ObjectPermissionsModel):
     )
     location = models.CharField(max_length=100)
     cover = FileField(related_name="event_covers")
-    groups_with_view_permission = models.ManyToManyField(AbakusGroup, blank=True)
+    can_view_groups = models.ManyToManyField(AbakusGroup, blank=True)
```

Comment:
This is already included in the ObjectPermissionModel this model inherits from, so you don't need to add this :smile:.

Reply:
Seems I was a bit too quick with just replacing groups_with_view_permission. Thanks for spotting it!